packages/ai/src/ai/modules/task/task_server.py

```
@@ -419,6 +420,14 @@ async def _dapbase_on_disconnected(self, conn: TaskConn) -> None:
                 # Log cleanup errors but continue processing other tasks
                 self.debug_message(f'Error during disconnection cleanup for task "{control.id}": {e}')
 
+        # Close any open file store handles for this connection
+        if hasattr(conn, '_account_info') and conn._account_info:
+            try:
+                fs = self.store.get_file_store(conn._account_info.clientid)
+                await fs.close_all_handles(connection_id)
+            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)